### ISSUE TITLE
idea: email sending for recurring invoices

### ISSUE THREAD

**Issue by TreyWW** [TreyWW/MyFinances#487]
### How big is this feature?

Medium

### Describe the problem

I'd like to have a way for users to configure email templates which allow them to automatically send emails to customers for events such as:
- invoice created
- invoice marked as paid
- invoice overdue

They should get a link to the invoice , via manage access.

**Comment by TreyWW** [comment]
/remind me 21h Take a look at this

**Comment by TreyWW** [comment]
/remind 21h Take a look at this

**Comment by TreyWW** [comment]
DELREPLY-2KD8NT9L

**Comment by TreyWW** [comment]
Added branch `feature/email_templates`, will work on after #478

**Comment by TreyWW** [comment]
`email_templates` on hold until the functionality of recurring invoice emails is complete. Although some work will need to be re-done it makes it a little easier to implement once a framework is in place.

**Comment by pvvramakrishnarao234** [comment]
If you are ok, i would like to work on this once the required framework is in place.

**Comment by TreyWW** [comment]
Hey @pvvramakrishnarao234, sure thing
